docs/architecture-decisions/0010-merge-cf-k8s-repositories.md

```diff
@@ -8,15 +8,15 @@ Accepted
 
 ## Context
 
-The `cf-k8s-api` and `cf-k8s-controllers` repositories were originally created separately to create a deliberate interface between "client" and "server" code. The intention was to motivate developers and users to reason about, change, and install the pieces separately. Instead, it has become difficult to decide which repository should house helper code and dependencies. Recently, we started working on a new `cf-on-k8s-integration` repository to pull the two components together and provide a home for both installation instructions and CI configuration for testing. As we worked on initialising this new repository, it became clear that different types of users wanted different things from it:
+The `cf-k8s-api` and `korifi` repositories were originally created separately to create a deliberate interface between "client" and "server" code. The intention was to motivate developers and users to reason about, change, and install the pieces separately. Instead, it has become difficult to decide which repository should house helper code and dependencies. Recently, we started working on a new `cf-on-k8s-integration` repository to pull the two components together and provide a home for both installation instructions and CI configuration for testing. As we worked on initialising this new repository, it became clear that different types of users wanted different things from it:
```

Comment:
I'm not sure if we should retroactively update these to change cf-k8s-controllers to korifi.

Reply:
Global search and replace :laughing: - will revert these changes.